README.rst

```diff
@@ -77,16 +77,22 @@ Get the code::
 
     git clone git@github.com:jazzband/django-floppyforms.git
     cd django-floppyforms
-    virtualenv -p python2 env
+    virtualenv env
     source env/bin/activate
     add2virtualenv .
 
 Install the development requirements::
 
     pip install "tox>=1.8"
 
+
+Currently, you'll need to `install the GeoDjango requirements`_ when running tests.
+
+:: _install the GeoDjango requirements: https://docs.djangoproject.com/en/3.0/ref/contrib/gis/install/geolibs/
```

Comment:
Ideally you could run most of the tests without this installed, to make sure our support for running the project without geodjango works.